ghc.nix

```diff
@@ -123,7 +124,7 @@ let
     ++ docsPackages
     ++ optional withLlvm llvmForGhc
     ++ optional withGrind valgrind
-    ++ optional withEMSDK emscripten
+    ++ optionals withEMSDK [ emscripten nodejs ]
     ++ optionals withWasm' [ wasi-sdk wasmtime ]
```

Comment:
It would be nice to also have this when `withWasm'` is true; since WASM JSFFI, certain tests (eg `jsffigc jsffioff jsffion jsffisleep`) also require it (also >=21).

Reply:
FTR: https://gitlab.haskell.org/ghc/ghc.nix/-/merge_requests/11